### PR TITLE
Feat: update README for furyctl-ng

### LINKED ISSUE #159: [Furyctl Next] Update README with basic instructions
The branch for Furyctl Next shall have its readme updated with the new features and usage instructions.

### PR DESCRIPTION
**Pull request by Al-Pragliola**
- Update furyctl's Readme with NG way of doing things with basic and advance usage instructions
- Improve help messages and some errors
- standardize spelling for furyctl

closes #159

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,27 +15,84 @@
...
+./dist/furyctl_darwin_amd64_v1/furyctl version
+```
+
+5. (optional) move the binary to your `bin` folder, in macOS:
+
+```console
+sudo mv ./dist/furyctl_darwin_amd64_v1/furyctl /usr/local/bin/furyctl
+```
+
+### Installation from binaries (not available yet for `furyctl-ng`)
```
Comment (nutellinoit) [inline README.md:88]: I would remove everything that is not inherent to the new furyctl version

File: README.md [hunk 2]
```diff
@@ -43,238 +101,228 @@ Alternatively, you can install `furyctl` using a brew tap or via an asdf plugin.
...
-Available Commands:
-  bootstrap   Creates the required infrastructure to deploy a battle-tested Kubernetes cluster, mostly network components
-  cluster     Creates a battle-tested Kubernetes cluster
-  completion  Generate completion script
-  help        Help about any command
-  init        Initialize the minimum distribution configuration
-  vendor      Download dependencies specified in Furyfile.yml
-  version     Prints the client version information
+```console
+furyctl create config --version <KFD version> --kind "EKSCluster"
```
Comment (nutellinoit) [inline README.md:166]: If `--distro-ocation` is mandatory, i would add that in all the example commands, for now

File: README.md [hunk 3]
```diff
@@ -43,238 +101,228 @@ Alternatively, you can install `furyctl` using a brew tap or via an asdf plugin.
...
-
-# The bases are a sets of Kustomize bases to deploy Kubernetes components
-bases:
-  - name: networking/
-  - name: monitoring/
-  - name: logging/
-  - name: ingress/
-  - name: dr/
-  - name: opa/
+1. The first phase, `infrastructure`, creates all the prerequisites needed to be able to create a cluster. For example, the VPC and its networks.
```
Comment (nutellinoit) [inline README.md:191]: Let's add somewhere that this phase will also connect the client to the VPN

File: README.md [hunk 4]
```diff
@@ -43,238 +101,228 @@ Alternatively, you can install `furyctl` using a brew tap or via an asdf plugin.
...
 
-The available `bootstrap` provisioners are:
+The available `infrastructure` provisioners are:
 
-| Provisioner | Description                                                                                                                                            |
-| ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `aws`       | It creates a VPC with all the requirements to deploy a Kubernetes Cluster. It also includes a VPN instance easily manageable by using `furyagent`.     |
+| Provisioner | Description                                                                                                                                        |
+| ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `aws`       | It creates a VPC with all the requirements to deploy a Kubernetes Cluster. It also includes a VPN instance easily manageable by using `furyagent`. |
```
Comment (nutellinoit) [inline README.md:317]: this seems to be related to the old furyctl

File: README.md [hunk 5]
```diff
@@ -43,238 +101,228 @@ Alternatively, you can install `furyctl` using a brew tap or via an asdf plugin.
...
 
-The available `cluster` provisioners are:
+The available `kubernetes` provisioners are:
 
-| Provisioner | Description                                                                                                                              |
-| ----------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
-| `eks`       | Creates an EKS cluster on an already existing VPC. It uses the [fury-eks-installer](https://github.com/sighupio/fury-eks-installer)      |
+| Provisioner | Description                                                                                                                         |
+| ----------- | ----------------------------------------------------------------------------------------------------------------------------------- |
+| `eks`       | Creates an EKS cluster on an already existing VPC. It uses the [fury-eks-installer](https://github.com/sighupio/fury-eks-installer) |
```
Comment (nutellinoit) [inline README.md:325]: this seems to be related to the old furyctl

**Review by nutellinoit — CHANGES_REQUESTED** [review]
Some comments, see conversations below 👇

File: README.md (hunk 1 above)
Comment (ralgozino) [reply README.md:88]: They will be relevant once we figure out how to manage releases in parallel or when we fully switch to ng.

If we delete these sections, we'll have to add them back soon™. But if they generate confusion let's delete them.

IMHO is acceptable to have a section that is not YET relevant for an alpha release. There's a warning in the header saying that you are viewing docs for an alpha version.

Comment (nutellinoit) [reply README.md:88]: from my point of view, IF we need some information about the old one, we can use a link to the old README pointing to a tag, and call it a day

Comment (ralgozino) [reply README.md:88]: This is not information about legacy, we'll use the same mechanism (and docs) for installing ng (bumping the version). These docs will be inherent to ng also, once we release it.

Comment (nutellinoit) [reply README.md:88]: OK

File: README.md (hunk 2 above)
Comment (ralgozino) [reply README.md]: Added the flag to all commands, it looks a little ugly but let's hope that we can remove it sooner than later.

File: README.md (hunk 3 above)
Comment (ralgozino) [reply README.md:192]: Added a note right before the `create cluster` command example:
https://github.com/sighupio/furyctl/blob/1a9794e5e3d9f24fa44a59dfc3906d33a85a10c4/README.md?plain=1#L210

File: README.md (hunk 4 above)
Comment (ralgozino) [reply README.md]: removed

File: README.md (hunk 5 above)
Comment (ralgozino) [reply README.md]: removed

**Review by omissis — APPROVED** [review]

File: .goreleaser.yml [hunk 6]
```diff
@@ -28,7 +28,7 @@ archives:
 checksum:
   name_template: 'checksums.txt'
 snapshot:
-  name_template: "{{ incpatch .Version }}-next"
+  name_template: "{{ incpatch .Version }}-ng"
```
Comment (nutellinoit) [inline .goreleaser.yml:31]: What is this change? will the binary be called furyctl-ng?

**Review by nutellinoit — APPROVED** [review]
LGTM with one question 👇

File: .goreleaser.yml (hunk 6 above)
Comment (ralgozino) [reply .goreleaser.yml:31]: TBD - this particular change is just to align the nomenclature. We'll address this when we'll think about the release process for ng vs legacy

**Pull request merged**